app/services/bulk_upload/processor.rb

```diff
@@ -144,4 +146,13 @@ def validator_class
       raise "Validator not found for #{bulk_upload.log_type}"
     end
   end
+
+  def handle_invalid_validator
+    if validator.errors.full_messages.include?("Template is blank - The template must be filled in for us to create the logs and check if data is correct.")
```

Comment:
Would it make sense to extract the error messages into a translation file? In case we change them!

Reply:
Sure, I can find out where they're set in the validator and use a translation here and there.